README.md

```diff
@@ -73,6 +73,22 @@ Currently only a 64-bit version is available.
 The Linux version does not currently automatically update so you will need to
 repeat these steps to upgrade to future releases.
 
+### Archive extraction
+
+An archive is available for people who don't want to install `atom` as root.
+
+This version enables you to install multiple atom versions in parallel. It has been built on Ubuntu 64-bits,
+but should be compatible with other Linux distributions.
+
+1. Install dependencies (on Ubuntu): `sudo apt install git gconf2 gconf-service libgtk2.0-0 libudev1 libgcrypt20
+libnotify4 libxtst6 libnss3 python gvfs-bin xdg-utils libcap2`
+2. Download `atom-amd64.tar.gz` from the [Atom releases page](https://github.com/atom/atom/releases/latest).
+3. Run `tar xf atom-amd64.tar.gz` in directory you want to extract atom folder.
```

Comment:
Something else I noticed on this line: it should probably be `in the directory where you want to extract the Atom folder`.